cis/libs/api.py

```diff
@@ -0,0 +1,53 @@
+import http.client
+import json
+import urllib
+
+
+class Person(object):
+    """Retrieve data from person api as needed."""
+    def __init__(self, person_api_config):
+        """
+        :param person_api_config, a dictionary of configuration information about how to connect to person_api
+        """
+
+        # Audience is either https://person-api.sso.mozilla.com or https://person-api.sso.allizom.org
+        self.audience = person_api_config.get('audience')
+        self.client_id = person_api_config.get('client_id')
+        self.client_secret = person_api_config.get('client_secret')
+        self.oidc_domain = person_api_config.get('oidc_domain')
```

Comment:
technically its oauth2 but not/without oidc in this case (nitpick)

Reply:
fixed